tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-perf-test-ci-pipeline.yml

```diff
@@ -0,0 +1,58 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_Training_Perf_Test
+
+  timeoutInMinutes: 120
+
+  variables: 
+  - group: 'ortperf' # variable group
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - script: >
+      tools/ci_build/github/linux/run_dockerbuild.sh
+      -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory)
+      -x "
+      --config RelWithDebInfo
+      --enable_training
+      --update --build
+      "
+    displayName: 'Build performance tests'
+
+  - script: >
+      docker run --gpus all -h OrtTrainingDev0 --rm --name onnxruntime-gpu-perf 
+      --volume $(Build.SourcesDirectory):/onnxruntime_src 
+      --volume $(Build.BinariesDirectory):/build 
+      --volume /bert_ort/bert_models:/build/bert_models:ro 
+      --volume /bert_data:/build/bert_data:ro 
+      -e NIGHTLY_BUILD onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6
+      /usr/bin/python3.6 "/onnxruntime_src/orttraining/tools/ci_test/run_bert_perf_test.py" 
```

Comment:
why are some of the strings in quotes?

Reply:
from other build pipeline example, I'll take out the quote and see